uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Service/Stats/StatsGatherer.php

```diff
@@ -263,4 +267,101 @@ private function isDrupal() {
     return $userSystem instanceof CRM_Utils_System_Drupal;
   }
 
+  /**
+   * Fetches entity counts for the job role entities
+   *
+   * @return array
+   */
+  private function getJobRoleEntityCounts() {
+    $recruitmentKey = 'com.civicrm.hrjobroles';
+    $counts = [];
+
+    if (!ExtensionHelper::isExtensionEnabled($recruitmentKey)) {
+      return $counts;
+    }
+
+    $counts['jobRole'] = $this->getEntityCount('HrJobRoles');
+    $params = ['option_group_id' => 'cost_centres'];
+    $counts['costCenter'] = $this->getEntityCount('OptionValue', $params);
+    $counts['funder'] = $this->getFunderCount();
+
+    return $counts;
+  }
+
+  /**
+   * Gets a count of all contacts in the system that have been marked as a
+   * funder
+   *
+   * @return int
+   */
+  private function getFunderCount() {
+    $result = civicrm_api3('HrJobRoles', 'get', ['return' => ['funder']]);
+    $result = array_column($result['values'], 'funder');
+    $funderIds = implode('', $result);
+    $funderIds = explode('|', $funderIds);
+    $funderIds = array_unique(array_filter($funderIds));
+
+    return count($funderIds);
+  }
+
+  /**
+   * Gets the count of custom groups and fields
+   *
+   * @return array
+   */
+  private function getCustomDataCounts() {
+    $counts = [];
+    $counts['customGroup'] = $this->getEntityCount('CustomGroup');
+    $counts['customField'] = $this->getEntityCount('CustomField');
+
+    return $counts;
+  }
+
+  /**
+   * Gets counts for the case entities
+   *
+   * @return array
+   */
+  private function getCaseEntityCounts() {
+    $counts = [];
+    $counts['caseType'] = $this->getEntityCount('CaseType');
```

Comment:
This will also include disabled Case Types. Is this expected?

Reply:
Good point, `getEntityCount` filters by deleted but it would be good to include `is_active` there too.